auditlog/models/log.py

```diff
@@ -31,7 +31,7 @@ def create(self, vals_list):
         for vals in vals_list:
             if not vals.get("model_id"):
                 raise UserError(_("No model defined to create log."))
-            model = self.env["ir.model"].browse(vals["model_id"])
+            model = self.env["ir.model"].sudo().browse(vals["model_id"])
```

Comment:
Why is sudo needed?

Reply:
Yes, in odoo 15 when refering to ir.* we need sudo